### PR TITLE
[paws] Document apis used in all collectors

### PR DESCRIPTION
**Pull request by surajkadam**

### CODE REVIEW

File: collectors/carbonblack/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "carbonblack-collector",
-  "version": "1.0.5",
+  "version": "1.0.6",
```
Comment (JDragovichAlertLogic) [inline collectors/carbonblack/package.json:3]: we doint need to bump the verison when we just do readme updates

File: collectors/sentinelone/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sentinelone-collector",
-  "version": "1.0.4",
+  "version": "1.0.5",
```
Comment (JDragovichAlertLogic) [inline collectors/sentinelone/package.json:3]: see previous comment

File: collectors/sophos/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sophos-collector",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (JDragovichAlertLogic) [inline collectors/sophos/package.json:3]: See previous comment

File: collectors/sophossiem/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sophossiem-collector",
-  "version": "1.0.1",
+  "version": "1.0.2",
```
Comment (JDragovichAlertLogic) [inline collectors/sophossiem/package.json:3]: see previous comment

File: collectors/sophossiem/README.md [hunk 5]
```diff
@@ -18,7 +18,12 @@ forward logs to the Alert Logic CloudInsight backend services.
 4. Click on Copy to copy your API Access URL and Headers from the API Token Summary section into your clipboard.<br />
 ![ScreenShot](./docs/img3.png)<br />
 
-### 2. CloudFormation Template (CFT)
+### 2. API Docs
+
+1. How to send alert and event data to your SIEM [Link](https://support.sophos.com/support/s/article/KB-000036372?language=en_US)
```
Comment (JDragovichAlertLogic) [inline collectors/sophossiem/README.md:23]: I would include instructions on how to access the swagger definition.

**Review by JDragovichAlertLogic — CHANGES_REQUESTED** [review]
Looks good so far suraj. Thanks

**Review by JDragovichAlertLogic — CHANGES_REQUESTED** [review]
Thanks Suraj. a few comments

File: collectors/o365/README.md [hunk 6]
```diff
@@ -29,7 +32,7 @@ In the Office 365 portal, you must register a new Office 365 web application to
 1. From the `All applications` tab on the `App registration (Preview)` blade, select `All apps`, and then click the application name you created. 
 1. Note the `Application ID`, for example, `a261478c-84fb-42f9-84c2-de050a4babe3`
 
-## Set Up the Required Active Directory Security Permissions
+### 2. Set Up the Required Active Directory Security Permissions
 
 1. On the main panel under the new application, select `View API Permissions`, and then click `+ Add`.
 1. Locate the `Office 365 Management APIs`, and then click `Select`.
```
[the remark this answers is not in the document]
Comment (surajkadam) [reply collectors/o365/README.md:38]: Yes, Sure Joe, But I need credentials for O365. I'll add screenshots as well if I get credentials.

**Review by JDragovichAlertLogic — APPROVED** [review]
Looks good. thanks for doing this Suraj

**Pull request merged**